back/engines/commercial/insights/app/finders/insights/inputs_finder.rb

```diff
@@ -20,7 +20,7 @@ def initialize(view, params = {})
     end
 
     def execute
-      inputs = view.scope.ideas
+      inputs = Idea.where(project_id: view.data_sources.select(:origin_id))
```

Comment:
Shouldn't you select with `where(origin_type: 'Project')`? I understand it doesn't matter right now, but just to make it consistent with the code base.